compiler/rustc_const_eval/src/interpret/memory.rs

```diff
@@ -855,6 +865,19 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         DumpAllocs { ecx: self, allocs }
     }
 
+    /// Print the allocation's bytes, without any nested allocations.
+    pub fn print_alloc_for_diagnostics(&self, id: AllocId) -> String {
```

Comment:
```suggestion
    pub fn print_alloc_bytes_for_diagnostics(&self, id: AllocId) -> String {
```
I was trying to figure out why this moved. I guess it now has to use `get_alloc_raw` to bypass the hook?